packages/framework/package.json

```diff
@@ -239,7 +239,6 @@
     "better-ajv-errors": "^1.2.0",
     "chalk": "^4.1.2",
     "cross-fetch": "^4.0.0",
-    "json-schema-faker": "^0.5.6",
```

Comment:
This package is causing a lot of ESM/CJS trouble. It was decided to hardcode its UMD version in the framework.